heed/src/env.rs

```diff
@@ -371,16 +373,15 @@ impl Drop for EnvInner {
 
 /// A helper function that transforms the LMDB types into Rust types (`MDB_val` into slices)
 /// and vice versa, the Rust types into C types (`Ordering` into an integer).
-extern "C" fn custom_key_cmp_wrapper<C: Comparator>(
+unsafe extern "C" fn custom_key_cmp_wrapper<C: Comparator>(
```

Comment:
Why have you added unsafe to this function? Isn't it safe now?

Reply:
It still dereferences raw pointers. Should I add a `/// # Safety` doc comment with some explanation? It is a private function though.

Reply:
Ho! Yeah! You're right. It's a good idea to add a safety note.

Reply:
Added the safety note